pom.xml

```diff
@@ -91,6 +91,9 @@
         <!-- Authorization Drools Policy Provider -->
         <version.org.drools>7.11.0.Final</version.org.drools>
 
+        <!-- Openshift -->
+        <version.com.openshift.openshift-restclient-java>6.1.3.Final</version.com.openshift.openshift-restclient-java>
```

Comment:
Did you check if this has been productized?

Reply:
Haven't looked. But it seems to be accordingly with ``da-cly`` results.

Reply:
@stianst, I forgot to add dependencies to server dist. PR updated. That is why tests were failing ...